lightning/src/ln/msgs.rs

```diff
@@ -1674,17 +1674,21 @@ pub use self::fuzzy_internal_msgs::*;
 #[cfg(not(fuzzing))]
 pub(crate) use self::fuzzy_internal_msgs::*;
 
+/// Bolt04 OnionPacket including hop data for the next peer
```

Comment:
nit: for consistency with the rest of the codebase -- 
```suggestion
/// BOLT 4 onion packet including hop data for the next peer.
```

Similar below, ending public doc sentences in periods